components/log-viewer-webui/client/src/pages/IngestPage/index.tsx

```diff
@@ -0,0 +1,30 @@
+import {
+    Col,
+    Row,
+} from "antd";
+
+import styles from "./index.module.css";
+import SpaceSavings from "./SpaceSavings";
+
+
+/**
+ * Presents compression statistics.
+ *
+ * @return
+ */
+const IngestPage = () => {
+    return (
+        <div className={styles["ingestPageGrid"]}>
+            <Row>
+                <Col span={8}>
+                    <SpaceSavings/>
+                </Col>
```

Comment:
Let's see the comment about
> It doesn't work when we add multiple of the cards, right?

I feel it's better to use `Col` as a parent component in the Stats component, assuming all our Stats panels are placed on this dashboard, which is currently a Row.
```suggestion
                <Col span={8}>
                    <SpaceSavings/>
                </Col>
```

Reply:
same got rid of antd grid...